Contrib/CLI/wcli.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+function config_extract() {
+    echo $(cat ~/.walletwasabi/client/Config.json | jq -r "$1")
+}
+
+ENABLED="$(config_extract '.JsonRpcServerEnabled')"
+CREDENTIALS=$(config_extract '.JsonRpcUser + ":" + .JsonRpcPassword')
+ENDPOINT=$(config_extract '.JsonRpcServerPrefixes[0]')
+BASIC_AUTH=$([ "$CREDENTIALS" == ":" ] && echo "" || echo "--user ${CREDENTIALS}")
+
+WALLETNAME=""
+
+if [  $# -ge 1 ]; then
+    ARG="${1%=*}"
+    if [[ "$ARG" == "-wallet" ]]; then
+        WALLETNAME="${1#*=}/"
+        shift
+    fi
+fi
+
+METHOD=$1
+shift
+
+if [ $# -ge 1 ]; then
+    if [[ "$1" ]]; then
+        PARAMS="\"$1\""
+        shift
+    else
+        PARAMS="\"\""
+        shift
+    fi
+
+    while (( "$#" )); do
+        if [[ "$1" ]]; then
+            PARAMS="$PARAMS, $1"
+        else
+            PARAMS='$PARAMS, ""'
```

Comment:
Shellcheck warning, I think it's error there - "[SC2016](https://www.shellcheck.net/wiki/SC2016) (info): Expressions don't expand in single quotes, use double quotes for that".

Reply:
removed